### ISSUE TITLE
text-align values (left, center and right) mistakenly marked as deprecated and non-standard?

### ISSUE THREAD

**Issue by datvm** [mdn/content#32545]
### MDN URL

https://developer.mozilla.org/en-US/docs/Web/CSS/text-align

### What information was incorrect, unhelpful, or incomplete?

See also [StackOverflow](https://stackoverflow.com/questions/78102109/is-css-text-align-center-really-deprecated-if-so-whats-the-alternative).

Right now there doesn't seem to be anything indicating the 3 values are deprecated. I think there is a mistake with [this commit change](https://github.com/mdn/content/commit/05a463a3bc1af6e1b1e0d6a273582d954ae00ed0).

### What did you expect to see?

The values `left`, `center` and `right` should not have deprecated and non-standard tags.

### Do you have any supporting links, references, or citations?

https://drafts.csswg.org/css-text/#text-align-property

### MDN metadata

<details>
<summary>Page report details</summary>

* Folder: `en-us/web/css/text-align`
* MDN URL: https://developer.mozilla.org/en-US/docs/Web/CSS/text-align
* GitHub URL: https://github.com/mdn/content/blob/main/files/en-us/web/css/text-align/index.md
* Last commit: https://github.com/mdn/content/commit/05a463a3bc1af6e1b1e0d6a273582d954ae00ed0
* Document last modified: 2024-02-21T08:43:26.000Z

</details>

**Comment by wbamberg** [comment]
Some background: the deprecated indicator is taken from the browser-compat-data, which [does indicate](https://github.com/mdn/browser-compat-data/blob/bfdddbf3857267b199aa0f518cd08917f2e24da0/css/properties/text-align.json#L101) that these values are deprecated.

What I think happened:
- in 2017, [the BCD for text-align was added](https://github.com/mdn/browser-compat-data/pull/621), and noted that [the **prefixed** block alignment values were deprecated](https://github.com/mdn/browser-compat-data/pull/621/files#diff-5bd1297dc8dc5a46d17410254db21cdc1e7aa8078e8f4311cd64d29d274221bfR56). This deprecation notice was taken from the [old, hardcoded BCD table](https://web.archive.org/web/20171026144338/https://developer.mozilla.org/en-US/docs/Web/CSS/text-align#Browser_compatibility) and duly made its way into the [new data-powered tables](https://web.archive.org/web/20181226052402/https://developer.mozilla.org/en-US/docs/Web/CSS/text-align#Browser_compatibility).

- then https://github.com/mdn/browser-compat-data/pull/20055 dropped the critical word "prefixed" and from then on the table was incorrect, but because the deprecation notice wasn't tied to a feature value, it was just a line in the BCD table, so wasn't obvious.

- then https://github.com/mdn/browser-compat-data/pull/22032 separated that line into actual feature values, and then the tooling started using that to add deprecation badges.

tldr: I think this bug report is right and those values should be un-deprecated.
